ethereum_common/src/ethereum_common/erc20_services.py

```diff
@@ -63,24 +63,24 @@ def approve_handler(m):
         rospy.Service('approve', Approve,
                       approve_handler)
 
-        def account_xrt_balance_handler(m):
+        def account_erc20_token_balance_handler(m):
             return AccountBalanceResponse(type_converter.strToUInt256(self.eth_utils.getTokenBalance(m.account.address)))
-        rospy.Service('account_xrt_balance', AccountBalance, account_xrt_balance_handler)
+        rospy.Service('account_erc20_token_balance', AccountBalance, account_erc20_token_balance_handler)
```

Comment:
This service name confuse me a little bit, may be just `erc20_balance` will be better?